ENTERPRISE-MIB

```diff
@@ -24,7 +24,7 @@ IMPORTS
            United Kingdom
            Tel: +44 20 8366 1177"
         DESCRIPTION  ""
-        ::= { dataConnection NUMBER }
+        ::= { dataConnection 9 }
```

Comment:
Doesn't this clash with https://github.com/Metaswitch/clearwater-snmp-handlers/blob/master/PROJECT-CLEARWATER-MIB#L36? Should you be choosing a new number (DOC-86854 in Metacom, I think)?